spec/fixtures/content_store/coronavirus_landing_page.json

```diff
@@ -392,19 +392,33 @@
       "list": [
         {
           "heading": "18 September",
+          "national_applicability": ["england", "northern_ireland", "scotland", "wales"],
           "paragraph": "If you live, work or travel in the North East, you need to [follow different covid rules](/guidance/north-east-of-england-local-restrictions)\n"
         },
         {
           "heading": "15 September",
+          "national_applicability": ["england"],
           "paragraph": "If you live, work or visit Bolton, you need to [follow different covid rules](/guidance/bolton-local-restrictions)\n"
         },
         {
           "heading": "14 September",
+          "national_applicability": ["scotland"],
           "paragraph": "People must not meet in groups larger than 6 in England. There are [exceptions to this 'rule of 6'](/government/publications/coronavirus-covid-19-meeting-with-others-safely-social-distancing/coronavirus-covid-19-meeting-with-others-safely-social-distancing#seeing-friends-and-family)\n"
         },
         {
           "heading": "24 July",
+          "national_applicability": ["wales"],
           "paragraph": "[Face coverings are mandatory in shops](/government/publications/face-coverings-when-to-wear-one-and-how-to-make-your-own/face-coverings-when-to-wear-one-and-how-to-make-your-own)\n"
+        },
+        {
+          "heading": "In Northern Ireland",
```

Comment:
It doesn't overly matter but might be good to change these heading values to be dates to match the ones before?

Reply:
Good point. Have updated and fixed up.

Reply:
👍